cfa_grub2.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = "cfa_grub2"
-  s.version     = "0.6.5"
+  s.version     = "0.6.6"
```

Comment:
all is fine except version. cfa follows semantic versioning https://semver.org/
and because this change is backward incompatible. Please use version 1.0.0.

Also please add to changes that change is backward incompatible and how to modify code to make it working ( expect array from `#terminal` and `#terminal=` requires array as argument.